### ISSUE TITLE
Unclear how to customize PR description in sl ghstack

### ISSUE THREAD

**Issue by discentem** [facebook/sapling#236]
Is it possible today to customize the PR description when submitting with `sl ghstack`? It would be really awesome to be able to customize the description with a template or even just append to the description.

**Comment by bolinfest** [comment]
I don't think this is really an option because ghstack has an existing logic that relies on parsing from the start of the pull request body. Changing the way the message is written would prevent ghstack from finding the other pull requests located in the stack.

If this were done at all, it would have to be in concert with upstream ghstack. /cc @ezyang
